### PR TITLE
Makes Poly less confused about the engine

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Added a few more lines for Poly to say, including a randomised delamination warning.
Fixed Poly complaining to engineering about the tesla engine, since it doesn't actually exist anymore.
Also changed the 'OH GOD WHY WOULD YOU PUT CARBON-DIOXIDE IN THE SM?' line to Plasma.

## Why It's Good For The Game

Poly is already abused enough, they should at least be able to give correct information.

## Changelog
:cl:
add: Added a few more lines for Poly to say.
tweak: Poly will no longer complain about the Tesla engine.
tweak: Changed the line about not putting CO2 into the SM to Plasma, since CO2 is generally safe.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 1]
```diff
@@ -685,13 +685,12 @@
...
 		"Poly wanna cracker!",
 		"Check the crystal, you chucklefucks!",
-		"Check the tesla, you shits!",
 		"STOP HOT-WIRING THE ENGINE, FUCKING CHRIST!",
 		"Wire the solars, you lazy bums!",
 		"WHO TOOK THE DAMN HARDSUITS?",
 		"OH GOD ITS ABOUT TO DELAMINATE CALL THE SHUTTLE",
 		"Why are there so many atmos alerts?",
-		"OH GOD WHY WOULD YOU PUT CARBON-DIOXIDE IN THE SM?",
+		"OH GOD WHY WOULD YOU PUT PLASMA IN THE SM?",
```
Comment (Kyep) [inline code/modules/mob/living/simple_animal/parrot.dm:693]:
```suggestion
		"OH GOD WHY WOULD YOU PUT PLASMA IN THE SM?",
		"Why is nobody watching the engine?",
		"Maybe the SM would produce more power if we fed it some clowns.",
		"Everyone else dusted when they touched the SM, but I am sure you will be different.",
		"I asked the mime if they turned off the scrubbers, but they didn't say a word.",
		"This engine setup meets all safety requirements.",
		"The natural diet of the SM consists of Chief Engineers.",
```

Comment (hal9000PR) [reply code/modules/mob/living/simple_animal/parrot.dm:693]: "don't eat the forbidden Nacho for fucks sake"

**Comment by Fox-McCloud** [comment]
I'd just change the CO2 reference and drop the Tesla reference and that's it--it keeps the lines direct and simple, as they were with the previous ones.

File: code/modules/mob/living/simple_animal/parrot.dm (hunk 1 above)
Comment (SabreML) [reply code/modules/mob/living/simple_animal/parrot.dm:693]: Wasn't planning on adding any more, but since you're got them all here...

**Review by Kyep — APPROVED** [review]

**Review by dearmochi — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**